src/main/java/com/pretz/dyntreeserver/service/dto/UserDTO.java

```diff
@@ -0,0 +1,56 @@
+package com.pretz.dyntreeserver.service.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class UserDTO {
+
+    private String name;
+
+    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
```

Comment:
what's this for?

Reply:
Without it, JSON parser ignored password field on both serialization and deserialization. In effect, it didn't read password parameter from JSON and what I got were domain objects with null password. Not pretty